ui/components/package.json

```diff
@@ -14,6 +14,11 @@
   "module": "dist/index.js",
   "main": "dist/cjs/index.js",
   "types": "dist/index.d.ts",
+  "sideEffects": [
```

Comment:
These files have side-effects in them because of what we're doing with `echarts.use` to keep echarts bundles small.